core/pkg/resultshandling/printer/v2/sarifprinter.go

```diff
@@ -116,12 +116,7 @@ func (sp *SARIFPrinter) printImageScan(scanResults *models.PresenterConfig) erro
 		return fmt.Errorf("no no image vulnerability data provided")
 	}
 
-	presenterConfig, err := presenter.ValidatedConfig(printer.SARIFFormat, "", false)
```

Comment:
was this accidentally removed?
cc @Daniel-GrunbergerCA

Reply:
No. Because with the new version of grype, the previous syntax is not supported. 
Similarly, in the `jsonprinter`, the `ValidatedConfig` method has been removed ([here](https://github.com/kubescape/kubescape/pull/1332/files#diff-6c52364e93973c10e45bf3a765f082f55e92004a744438374dd212f188c4ff8aR81))